src/database/mod.rs

```diff
@@ -381,6 +381,15 @@ impl PullRequestModel {
             ApprovalStatus::NotApproved => None,
         }
     }
+
+    /// Approved but with a failed auto build.
+    pub fn stalled(&self) -> bool {
```

Comment:
We should document this in one of the design/doc markdown files, but it doesn't need to happen in this PR, we are kind of lagging with updating these doc files anyway.